README.md

```diff
@@ -448,14 +448,24 @@ git clone git@github.com:kethinov/teddy.git
 cd teddy
 ```
 
-Install dependencies for the server test:
+Install dependencies for the tests:
 
 ```
 npm i
 ```
 
-Run tests:
+For client side tests you'll need the browser you want to test on installed. (Chrome is default)
+
+Run tests (server and chrome:
 
 ```
 npm test
 ```
+
+Other browsers:
```

Comment:
Change to "The default `npm test` command runs the tests server-side in Node.js and client-side in Chrome. See below commands for other browsers:"